src/greatwall/gui.py

```diff
@@ -46,7 +54,7 @@ def run(self):
                     f"GreatWall initialization doesn't match the current level {self.greatwall.current_level}"
                 )
             elif self._is_initializing and not self.greatwall.current_level:
-                self.greatwall.initialize_state_hashes()
```

Comment:
Refactored?

Reply:
Its internal implementation does mot changed, just its name has been changed to match our convention in naming in the whole project.